card/lib/card.rb

```diff
@@ -47,6 +47,7 @@ def self.serializable_attr_accessor *args
     :comment, :comment_author,    # obviated soon
     :update_referencers,          # wrong mechanism for this
     :update_all_users,            # if the above is wrong then this one too
+    :silent_change,               # and this probably too
```

Comment:
I'll merge this.  Will be good to have a grand API discussion after the Post-it Summit :)